docs/getting-started/devenv/using-visual-studio.md

```diff
@@ -32,11 +32,11 @@ Visual Studio 2015 provides a full-featured development environment for developi
 
 The following steps will set up Visual Studio 2015 for .NET Core development:
 
-1. Verify that you're running Visual Studio 2015 Update 2:
+1. Verify that you're running Visual Studio 2015 Update 3:
```

Comment:
Do we really need this here since the Update 3 is already listed in the prerequisites? I think it would be better to just say Open Visual Studio in this step.